understanding/20/keyboard.html

```diff
@@ -50,8 +50,7 @@ <h2>Intent of Keyboard</h2>
          example, clicking, selecting, moving, sizing). However, there is a small class of
          input that is done with a pointing device that cannot be done from the keyboard in
          any known fashion without requiring an inordinate number of keystrokes. Free hand
-         drawing, watercolor painting, and flying a helicopter through an obstacle course are
-         all examples of functions that require path dependent input. Drawing straight lines,
+         drawing, or watercolor painting require path dependent input. Drawing straight lines,
```

Comment:
thank you @chaals for removing the peculiar helicopter example there...hadn't even clocked it at the time

Reply:
@patrickhlauke the helicopter example was meant to be a gaming reference.

Reply:
sure, but not really representative, since there's a myriad of games where you can control a helicopter with keyboard, joystick, etc.